daemon/loop.go

```diff
@@ -150,6 +151,49 @@ func (d *Daemon) pullAndSync(logger log.Logger) {
 		}); err != nil {
 			logger.Log("err", err)
 		}
+
+		// Find notes in revisions.
+		for _, rev := range revisions {
```

Comment:
Will these be in the right order? In general we'd want notifications from earliest to latest.